app/helpers/application_widgets_helper.rb

```diff
@@ -0,0 +1,43 @@
+module ApplicationWidgetsHelper
+  # Create a edit button with icon
+  #
+  # @param options [String, Hash] The url or the url options which will be passed to `link_to`.
```

Comment:
I'm using options as the first param's name now, same as https://github.com/rails/rails/blob/14d0e0702651de4d825769bddd574aa5066e86f3/actionview/lib/action_view/helpers/url_helper.rb#L180

What do you think now ?